airbyte-integrations/connectors/source-bing-ads/acceptance-test-config.yml

```diff
@@ -1,26 +1,41 @@
 connector_image: airbyte/source-bing-ads:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_bing_ads/spec.json"
-  connection:
-    - config_path: "secrets/config_old.json"
-      status: "succeed"
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+    - spec_path: source_bing_ads/spec.json
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+    - config_path: secrets/config.json
+  connection:
+    tests:
+    - config_path: secrets/config_old.json
+      status: succeed
+    - config_path: secrets/config.json
+      status: succeed
+    - config_path: integration_tests/invalid_config.json
+      status: failed
   basic_read:
-    - config_path: "secrets/config.json"
-      timeout_seconds: 600
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: ["account_performance_report_hourly","ad_group_performance_report_hourly","ad_performance_report_hourly","campaign_performance_report_hourly","keyword_performance_report_hourly"]
-  # incremental test doesn"t work if a single stream has multiple states
-  #incremental:
-  #  - config_path: "secrets/config.json"
-  #    configured_catalog_path: "integration_tests/configured_catalog.json"
-  #    future_state_path: "integration_tests/abnormal_state.json"
+    tests:
+    - config_path: secrets/config.json
+      expect_records:
+        path: "integration_tests/expected_records.txt"
+        extra_records: yes
+      empty_streams:
+      - name: account_performance_report_hourly
+        bypass_reason: "Hourly reports are disabled, because sync is too long"
+      - name: ad_group_performance_report_hourly
+        bypass_reason: "Hourly reports are disabled, because sync is too long"
+      - name: ad_performance_report_hourly
+        bypass_reason: "Hourly reports are disabled, because sync is too long"
+      - name: campaign_performance_report_hourly
+        bypass_reason: "Hourly reports are disabled, because sync is too long"
+      - name: keyword_performance_report_hourly
+        bypass_reason: "Hourly reports are disabled, because sync is too long"
   full_refresh:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
+    tests:
+    - config_path: secrets/config.json
+      configured_catalog_path: integration_tests/configured_catalog.json
+  incremental:
+    tests:
+    bypass_reason: "SAT doesn't support complex nested states used in incremental report streams"
```

Comment:
This is interesting! Can you share the example?

Reply:
stream returns reports for each ads campaign, so we need to keep a state for each campaign.

Here is an example:

[
    {
        "type": "STREAM",
        "stream": {
            "stream_descriptor": {
                "name": "campaign_performance_report_daily"
            },
            "stream_state": {
                "180278106": {
                    "TimePeriod": 1627948800
                },
                "18027999": {
                    "TimePeriod": 1627948800
                },
                "... another account": {
                    "TimePeriod": 1627948800
                }
            }
        }
    }
]

Reply:
Ah, the "TimePeriod" is changing I bet.  OK